geckomat/model_adapter/HumanGEMAdapter.m

```diff
@@ -42,8 +52,33 @@
 			rxns_tsv = importTsvFile(strcat(getHumanGEMRootPath(),'model/reactions.tsv'));
 			spont = rxns_tsv.spontaneous;
 			spontRxnNames = rxns_tsv.rxns;
-		end
+        end
 		
+        %Ensembl gene ids are not available in uniprot (ensembl returns transcripts, not genes)
+        %Therefore, we collect gene symbols from uniprot, and need to convert the ensembl genes
+        %here to gene symbols as well
+        function genes = getUniprotCompatibleGenes(obj,model)
+            % get the path
+            tmpfile = fullfile(getHumanGEMRootPath(),'model','genes.tsv');
+
+            % import as structure, convert to table, and extract header
+            tmp = struct2table(importTsvFile(tmpfile));
```

Comment:
No major comment, as this only affects human-GEM. But these lines (`genes.tsv` file and `importTsvFile` function) require the full human-GEM repository to be present (and in MATLAB path). This should be clearly communicated before/when using this adapter, as users might just come with the human-GEM.xml file instead.

Reply:
I have always assumed that the whole Human-GEM is available for everything I do. Sure, this needs to be communicated.